ngraph_creator/src/OperationsFactory.cpp

```diff
@@ -123,6 +123,8 @@ std::shared_ptr<OperationsBase> OperationsFactory::getOperation(
             return std::make_shared<Reshape>(operationIndex);
         case OperationType::RNN:
             return std::make_shared<RNN>(operationIndex);
+		case OperationType::ROI_POOLING:
+            return std::make_shared<ROI_Pooling>(operationIndex);
```

Comment:
Coding style check??

Reply:
I think it was because of rebase, will update it.